graphql/schema.graphql

```diff
@@ -335,9 +335,9 @@ type Host {
 }
 
 type InstanceTag {
-  key: String
-  value: String
-  canBeModified: Boolean
+  key: String!
+  value: String!
+  canBeModified: Boolean!
 }
```

Comment:
What would happen if a user tries to delete a tag by editing the tag to be "" : ""? Will they get an error? Is that what we want? (I'm guessing yes).

Reply:
As far as i can tell there isn't any validation of empty tags within the evergreen app code. It looks like empty tags are supported on the aws side of things. 
![image](https://user-images.githubusercontent.com/4605522/102100688-bbafa900-3df7-11eb-927d-0b38b774794b.png)

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html